lib/services/profiles-api-service.js

```diff
@@ -315,11 +326,12 @@ function profileApiServiceFactory(
                                 'Unable to retrieve valid node bootSettings', node.type);
                     }
                 } else {
-                    return self._handleProfileRenderError(
-                        'Unable to locate active workflow or there are no bootSettings',
-                        node.type,
-                        400
-                    );
+                    return {
+                        profile: 'ipxe-info.ipxe',
```

Comment:
it keeps the code logic with before. not sure about original switch purpose . from original code logic  perspective, I think it's fine here, if any concerns, let me update